Lua/Lua.sublime-syntax

```diff
@@ -9,82 +9,29 @@ file_extensions:
 
 first_line_match: |-
   (?xi:
-    ^ \#! .* {{shebang_language}}           # shebang
-  | ^ \s* -- .*? -\*- .*? \blua\b .*? -\*-  # editorconfig
+    ^ \#! .* {{first_line_language}}                                            # shebang
```

Comment:
`shebalng_language` is what is used in all other syntaxes and so Lua should keep doing so.